src/EnergyPlus/AirflowNetworkBalanceManager.cc

```diff
@@ -4155,6 +4398,23 @@ namespace AirflowNetworkBalanceManager {
 		static bool OneTimeFlag( true );
 		static bool ErrorsFound( false );
 		Real64 GlobalOpenFactor;
+		Real64 ZonePressure1;
```

Comment:
@lgu1234 @Myoldmopar Maybe there is no reason to be concerned, but I try to set an initial value in the declaration just in case there's a potential path through the if logic that results in using the variable uninitialized (leading to the cursed NANs possibly showing up somewhere else down the line).  I don't think the compilers can know if that might happen.  In this case, it's probably fine as-is.